worker/src/lib/HeliconeProxyRequest/forwarder.ts

```diff
@@ -73,13 +75,33 @@ export async function proxyForwarder(
   }
 
   if (cacheSettings.shouldReadFromCache) {
+    const { data: auth, error: authError } = await request.auth();
+    if (authError == null) {
+      const db = new DBWrapper(env, auth);
+      const { data: orgData, error: orgError } = await db.getAuthParams();
```

Comment:
Can we move the getAuthParams to the `request` object instead. This way we don't need to create a DB wrapper?

Reply:
you can do this in a follow up PR too

Reply:
I will pair-code with Cole to make this in the feature.